cortex-mixin/alerts/compactor.libsonnet

```
@@ -6,11 +6,9 @@
         {
           // Alert if the compactor has not successfully cleaned up blocks in the last 24h.
           alert: 'CortexCompactorHasNotSuccessfullyCleanedUpBlocks',
-          'for': '15m',
+          'for': '1h',
           expr: |||
             (time() - cortex_compactor_block_cleanup_last_successful_run_timestamp_seconds > 60 * 60 * 24)
```

I think we can lower the threshold to 6 hours. I've checked our infra and a 6h threshold looks large and good enough.

```suggestion
            (time() - cortex_compactor_block_cleanup_last_successful_run_timestamp_seconds > 60 * 60 * 6)
```

**If you pick up this suggestion, we need to update the `message` below too.**